README.md

```diff
@@ -99,20 +86,56 @@ Note that you may not get cached results if you use a different `nix` version th
 
 ## Building arbitrary packages
 
-The [`survey`](./survey) directory maintains a select set of Haskell executables that are known and not known to work with this approach; contributions are welcome to grow the set of working executables.
-Run for example:
+The [`survey/default.nix`](./survey/default.nix) was originally a survey of Haskell executables that are known to (and known _not_ to) work with this approach, however it also exposes packages sets with overridden Haskell packages and dependencies that you can use to build _your own_ packages. The name `survey` shouldn't put you off from using it.
+
+If you are a nix user, you can `import` this functionality and override the `haskellPackages` to include your own package, for example in the [PostgREST project](https://github.com/PostgREST/postgrest/blob/main/nix/static-haskell-package.nix).
```

Comment:
The PostgREST example is slightly complex, but I think it's probably better to include an (recent and maintained) example rather than not though?

Reply:
I think PostgREST ist fine but we should pin th link to a git commit (key <kbd>Y</kbd> in Github).